packages/tasks/src/local-apps.ts

```diff
@@ -200,6 +217,13 @@ export const LOCAL_APPS = {
 		displayOnModelPage: isLlamaCppGgufModel,
 		snippet: snippetLlamacpp,
 	},
+	"node-llama-cpp-cli": {
+		prettyLabel: "node-llama-cpp (CLI)",
+		docsUrl: "https://node-llama-cpp.withcat.ai",
+		mainTask: "text-generation",
```

Comment:
I think we can use the `conversational` tage here: https://huggingface.co/models?other=conversational

cc: @mishig25 WDYT?

Reply:
`conversational` is not a task, it's a sub-task. I would expect this to be consistent with llama.cpp snippet, which uses `text-generation`